frontend/src/app/Settings/Facility/Components/OrderingProvider.tsx

```diff
@@ -105,66 +109,103 @@ const OrderingProvider: React.FC<Props> = ({
+            errors?.orderingProvider?.NPI?.type ? "error" : undefined
           }
-          errorMessage={errors["orderingProvider.NPI"]}
+          errorMessage={errors?.orderingProvider?.NPI?.message}
         />
         <TextInput
           label="Phone number"
           name="op-phone"
           required={isRequired}
-          value={provider.phone || ""}
-          onChange={onChange}
-          onBlur={() => {
-            validateField("orderingProvider.phone");
-          }}
+          value={formCurrentValues.orderingProvider.phone ?? undefined}
           validationStatus={
-            errors["orderingProvider.phone"] ? "error" : undefined
+            errors?.orderingProvider?.phone?.type ? "error" : undefined
           }
-          errorMessage={errors["orderingProvider.phone"]}
+          errorMessage={errors?.orderingProvider?.phone?.message}
+          registrationProps={register("orderingProvider.phone", {
+            required: isRequired ? "Ordering provider phone is required" : "",
+            validate: {
+              validPhone: (opPhone: string) =>
+                opPhone?.length
+                  ? phoneNumberIsValid(opPhone) ||
+                    "Ordering provider phone number is invalid"
+                  : true,
+            },
+          })}
         />
         <TextInput
           label="Street address 1"
           name="op-street"
-          value={provider.street || ""}
-          onChange={onChange}
+          value={formCurrentValues.orderingProvider.street ?? undefined}
+          registrationProps={register("orderingProvider.street", {
+            required: false,
+          })}
         />
         <TextInput
           label="Street address 2"
           name="op-streetTwo"
-          value={provider.streetTwo || ""}
-          onChange={onChange}
+          value={formCurrentValues.orderingProvider.streetTwo ?? undefined}
+          registrationProps={register("orderingProvider.streetTwo", {
+            required: false,
+          })}
         />
         <TextInput
           label="City"
           name="op-city"
-          value={provider.city || ""}
-          onChange={onChange}
+          value={formCurrentValues.orderingProvider.city ?? undefined}
+          registrationProps={register("orderingProvider.city", {
+            required: false,
+          })}
         />
         <TextInput
           label="ZIP code"
           name="op-zipCode"
-          value={provider.zipCode || ""}
-          onChange={onChange}
+          value={formCurrentValues.orderingProvider.zipCode ?? undefined}
+          validationStatus={
+            errors?.orderingProvider?.zipCode?.type ? "error" : undefined
+          }
+          errorMessage={errors?.orderingProvider?.zipCode?.message}
+          registrationProps={register("orderingProvider.zipCode", {
+            required: false,
+            validate: {
+              validZip: (zipCode: string) =>
```

Comment:
⚠️ decided against adding if the zip code was valid for the state validation here because we were making too many requests and getting back 429 errors from SmartyStreets 😅 -- so like it was done before, checking if the zip code and state combination are good only on submit after the form passes initial validation check